.github/workflows/pr-automation.yml

```
@@ -133,3 +134,12 @@ jobs:
       spec: ${{ needs.parse-tags.outputs.spec}}
       matrix: ${{ needs.parse-tags.outputs.matrix}}
       is-pg-build: ${{ github.event.pull_request.base.ref == 'pg' }}
+
+  perform-it:
+    needs: [ parse-tags, perform-test ]
+    if: success() && needs.parse-tags.outputs.its == 'true'
+    uses: ./.github/workflows/server-integration-tests.yml
+    secrets: inherit
+    with:
+      pr: ${{ github.event.number }}
+      is-pg-build: ${{ github.event.pull_request.base.ref == 'pg' }}
```

actionlint (1.7.4)

input "is-pg-build" is typed as string by reusable workflow "./.github/workflows/server-integration-tests.yml". bool value cannot be assigned

(expression)